tests/src/runtimeApi/module/hipFuncGetAttributes.cpp

```diff
@@ -18,7 +18,7 @@ THE SOFTWARE.
 */
 
 /* HIT_START
- * BUILD: %t %s ../../test_common.cpp NVCC_OPTIONS -std=c++11
+ * BUILD: %t %s ../../test_common.cpp NVCC_OPTIONS -std=c++11 EXCLUDE_HIP_PLATFORM clang
```

Comment:
Why are we skipping these tests? they should be passing in HIP-Clang.

Reply:
These tests currently fail when compiled with HIP-clang compiler with HIP-HCC runtime. Even though testcases are shared , HIT environment is not being using in HIP-VDI. Disabled tests temporarily on HIP-HCC path.